goal_src/jak2/levels/ruins/ruins-scenes.gc

```diff
@@ -1355,7 +1355,7 @@
     (the-as pair 0)
     )
   (set! (-> obj part) (create-launch-control (-> *part-group-id-table* 78) obj))
-  (minimap-method-12 *minimap* obj (the-as uint 16) (the-as int #f) (the-as vector #t) 0)
+  ;; (minimap-method-12 *minimap* obj (the-as uint 16) (the-as int #f) (the-as vector #t) 0)
```

Comment:
it might be better to leave this, then make an empty `minimap-method-12` in `minimap.gc`, so we don't have to remember to come back and add these all back.